libibverbs/verbs.c

```diff
@@ -200,6 +200,22 @@ LATEST_SYMVER_FUNC(ibv_query_pkey, 1_1, "IBVERBS_1.1",
 	return 0;
 }
 
+LATEST_SYMVER_FUNC(ibv_get_pkey_index, 1_1, "IBVERBS_1.1",
```

Comment:
libibverbs/libibverbs.map.in needs to be updated with this symbol, and the version should be IBVERBS_1.2
Same for debian/debian/libibverbs1.symbols

CI will fail because of this.

Reply:
From libibverbs.map.in: /* NOTE: The next stanza for public symbols should be IBVERBS_1.4 due to release 12 */. I'm confused ...

Reply:
I'm not familiar with rdma-core shared library versioning. Can you have a look at the update I pushed out?

Reply:
Looks like the note in the map file is almost correct (woops), release 12 removed the IB_VERBS_1.3 and 1.4 stanza's.

So the next lowest unused number is IBVERBS_1.5, which is what you should use here.